### PR TITLE
Archimate extension

### PR DESCRIPTION
**Pull request by stefjoosten**
.xml-files are interpreted as ArchiMate repository.

### CODE REVIEW

**Comment by hanjoosten** [comment]
This pull request should close issue #1027

File: ArchiExtension/archiAnalyze/stack.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# This file was automatically generated by stack init
```
Comment (hanjoosten) [inline ArchiExtension/archiAnalyze/stack.yaml]: This file should be removed!

File: ArchiExtension/archiAnalyze/archiAnalyze.cabal [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name:                archiAnalyze
+version:             0.1.0.0
+synopsis:            Simple project template from stack
+description:         Please see README.md
+homepage:            https://github.com/githubuser/archiAnalyze#readme
```
Comment (hanjoosten) [inline ArchiExtension/archiAnalyze/archiAnalyze.cabal]: All files in ArchiExtension/archiAnalyze/* should be removed. They are obsolete.

File: src/Ampersand/Input/Archi/ArchiAnalyze.hs [hunk 3]
```diff
@@ -0,0 +1,705 @@
...
+              }
+data P_NamedRel = PNamedRel { p_nrpos :: Origin, p_nrnm :: String, p_mbSign :: Maybe P_Sign }
+
+data P_Sign = P_Sign {pSrc :: P_Concept, pTgt :: P_Concept } deriving (Ord,Eq)
+data PAtomPair
+  = PPair { pppos :: Origin
+          , ppLeft  :: PAtomValue
+          , ppRight :: PAtomValue
+          } deriving (Eq,Ord,Show) -- Show is for QuickCheck error messages and/or input redundancy removal only!
+-}
```
Comment (hanjoosten) [inline src/Ampersand/Input/Archi/ArchiAnalyze.hs]: Please remove this, get used to use the Haddock generated index file for this kind of info

File: src/Ampersand/Input/Archi/ArchiAnalyze.hs [hunk 4]
```diff
@@ -0,0 +1,705 @@
...
+        = ( pPp{p_popas = p_popas pPp `union` p_popas pPp'}
+          , case (mDecl, mDecl') of
+                 (Just decl, Just _)     -> Just (decl {dec_nm = dec_nm decl++"!"})
+                 (Just decl, Nothing)    -> Just (decl {dec_nm = dec_nm decl++"!"})
+                 (Nothing,   Just decl') -> Just (decl'{dec_nm = dec_nm decl'++"!"})
+                 _                       -> Nothing
+          , pgns `union` pgns'
+          )
+       _ `squash` _ = fatal "error in squash"
+-}
```
Comment (hanjoosten) [inline src/Ampersand/Input/Archi/ArchiAnalyze.hs]: Is this code still of any value? If not, please remove it.

File: src/Ampersand/Input/Archi/ArchiAnalyze.hs (hunk 3 above)
Comment (stefjoosten) [reply src/Ampersand/Input/Archi/ArchiAnalyze.hs]: done....

File: src/Ampersand/Input/Archi/ArchiAnalyze.hs (hunk 4 above)
Comment (stefjoosten) [reply src/Ampersand/Input/Archi/ArchiAnalyze.hs]: done...

**Comment by hanjoosten** [comment]
Thanks for the PR!

**Pull request merged**